XenModel/Actions/Updates/DownloadUpdatesXmlAction.cs

```diff
@@ -254,12 +254,14 @@ private void GetXenServerVersions(XmlDocument xdoc)
                     }
 
                     List<XenServerPatch> patches = new List<XenServerPatch>();

```

Comment:
Should we be doing the same with patches? What would happen if the patch node were absent?